projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-suggestions-from-ai/index.ts

```diff
@@ -88,7 +88,10 @@ export default function useSuggestionsFromAI( {
 	 * @returns {void}
 	 */
 	const handleSuggestion = useCallback(
-		( event: CustomEvent ) => onSuggestion( event?.detail ),
+		( event: CustomEvent ) => {
+			// Remove the delimiter from the suggestion.
+			onSuggestion( event?.detail?.replaceAll( delimiter, '' ) );
```

Comment:
do we need to remove it from the AI response? What happens if the content contains the same chars that the delimiter?

Reply:
The `before` screenshot has the delimiter in the AI response.

>What happens if the content contains the same chars that the delimiter?

It would be lost, that is why I changed it from `####`, as 4 backticks should be very rare. We could change it to something like `!?;` or something.